src/global/reco/reco.cc

```diff
@@ -305,5 +305,37 @@ void InitPlugin(JApplication *app) {
     ));
 #endif
 
+    app->Add(new JOmniFactoryGeneratorT<TransformBreitFrame_factory>(
+            "GeneratedBreitFrameParticles",
+            {"MCParticles","InclusiveKinematicsElectron","GeneratedParticles"},
+            {"GeneratedBreitFrameParticles"},
+            {},
+            app
+    ));
+
+    app->Add(new JOmniFactoryGeneratorT<JetReconstruction_factory<edm4eic::ReconstructedParticle>>(
+            "GeneratedCentauroJets",
+            {"GeneratedBreitParticles"},
```

Comment:
```suggestion
            {"GeneratedBreitFrameParticles"},
```

Reply:
Dammit. Spent 20 minutes debugging this when I should have just read @veprbl 's review. Good catch!